src/legacy/core_plugins/interpreter/public/functions/tagcloud.js

```diff
@@ -31,13 +31,63 @@ export const tagcloud = () => ({
     defaultMessage: 'Tagcloud visualization'
   }),
   args: {
-    visConfig: {
-      types: ['string', 'null'],
-      default: '"{}"',
+    scale: {
+      types: ['string'],
+      default: 'linear',
+    },
+    orientation: {
+      types: ['string'],
+      default: 'single',
+    },
+    minFontSize: {
+      types: ['number'],
+      default: 18,
+    },
+    maxFontSize: {
+      types: ['number'],
```

Comment:
should we (and how) provide minimum and maximum for this property ?

Reply:
@w33ble ^^ whats the canvas way ?

Reply:
Much like options, the interpreter doesn't care. Check it in the function, if the wrong value is provided, just throw.